.Gruntfile.babel.js

```diff
@@ -107,15 +106,9 @@ module.exports = (grunt) => {
       },
       assets: {
         cwd: 'frontend/assets',
-        src: ['**/*', '!sass/**'],
+        src: ['**/*', '!style/**'],
         dest: distAssets,
         expand: true
-      },
-      fontawesome: {
-        cwd: 'node_modules/@fortawesome/fontawesome-free/webfonts/',
-        src: ['fa-regular*', 'fa-solid*'],
-        dest: `${distAssets}/fonts`,
-        expand: true
```

Comment:
Did you remove this because you copied the raw font awesome files into the project?

If so, you should delete `@fortawesome/fontawesome-free` from `package.json` since you're not using it.

However, I personally prefer not manually adding the font awesome files to the repo, because the way we have it currently allows us to stay up-to-date with the latest font-awesome icons, which are updated to add support for new services that come along, etc.

So my recommendation is to revert the font-awesome changes, unless you have a good reason not to, in which case I'd like to hear it!

Reply:
We use 5% of the font at the moment and we don't update font with fontawesome update but with designer update. So I plan to create our own font using https://icomoon.io that include all the svg that we use in the project. It's very easy to replace icons or add icons as we want.

Reply:
OK, sounds good. Can you update the style guide to mention how to do that? (Or put a note / TODO somewhere?)